code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/deployment/impl/DefaultDeploymentUnitInstance.java

```diff
@@ -94,19 +96,17 @@ public DeploymentUnitInstance start(List<Integer> volumesFromInstancesIds) {
             this.instance = instanceMapPair.getLeft();
             this.exposeMap = instanceMapPair.getRight();
         }
-        DeferredUtils.nest(new Runnable() {
-            @Override
-            public void run() {
-                context.objectProcessManager.scheduleStandardProcess(StandardProcess.CREATE, instance, null);
-                context.objectProcessManager.scheduleStandardProcess(StandardProcess.CREATE, exposeMap, null);
-            }
-        });
+        context.objectProcessManager.scheduleStandardProcessAndPublishEvent(StandardProcess.CREATE, instance,
+                null);
+        context.objectProcessManager.scheduleStandardProcessAndPublishEvent(StandardProcess.CREATE, exposeMap,
+                null);
+
         if (InstanceConstants.STATE_STOPPED.equals(instance.getState())) {
             DeferredUtils.nest(new Callable<Object>() {
                 @Override
                 public Object call() throws Exception {
-                    context.objectProcessManager.scheduleProcessInstance(InstanceConstants.PROCESS_START, instance,
-                            null);
+                    context.objectProcessManager.scheduleProcessInstanceAndPublishEvent(
```

Comment:
Since this new call: scheduleProcessInstanceAndPublishEvent() already nests the process, do we need this outer one (line 105)?